### PR TITLE
Webdriver to nightwatch

### PR DESCRIPTION
**Pull request by NowTV-Assurance**
This branches off the fix-docs branch to fix fix tests, it also now runs the toolkit "functional/feature" tests on multiple browsers.

- added a node hook so docs server doesn't attempt to import sass files
- removed references to ./dist from all docs code
  - before this, it was requiring in 2 instances of React, making it impossible to use refs (one from each webpack bundle)
- added loaders from toolkit build into docs build so docs client can handle the sass, svgs and fonts
- removed webdriverIO
- added nightwatch with configurable args
- removed phantomjs
- removed selenium-standalone and replaced with .jar that is automatically started by nightwatch
- added more feature tests for accordi

### CODE REVIEW

**Comment by alexjfno1** [comment]
:+1:

File: test/config/nightwatch.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      },
+      "desiredCapabilities": {
+        "project": "nowtv-Toolkit",
+        "build": "build local",
+        "browserName": "firefox",
+        "javascriptEnabled": true,
+        "acceptSslCerts": true,
+        "browserstack.local" : true,
+        "browserstack.user": "john.sibly@sky.uk",
+        "browserstack.key": "tqpqbaVxZoFzHTvRWU4R"
```
Comment (nowarkdev) [inline test/config/nightwatch.json]: PO: This is a public repo, I think this needs to be scrubbed from the Git history.

**Comment by nowarkdev** [comment]
PO: Looking good, could we get a bit of an addition to the readme on any potential setup/usage?

**Comment by nowarkdev** [comment]
PO: :+1:

**Pull request merged**